.github/ISSUE_TEMPLATE/release_tracking.md

```diff
@@ -0,0 +1,77 @@
+---
+name: 🚋 Release cycle tracking
+about: Create a new release cycle tracking issue for a minor release
+title: Tasks for v<release-tag> release cycle
+labels: ''
+assignees: ''
+
+---
+
+Please see the corresponding section in [release-tasks.md](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md) for documentation of individual tasks.  
+
+## Tasks
+
+**Notes**:
+* Weeks are only specified to give some orientation.
+* The following is based on the v1.4 release cycle. Modify according to the tracked release cycle.
+
+Week -3 to 1:
+* [ ] [Release Lead] Set a tentative release date for the minor release
+* [ ] [Release Lead] Assemble release team
+
+Week 1:
+* [ ] [Release Lead] Finalize release schedule and team
+* [ ] [Release Lead] Prepare main branch for development of the new release
```

Comment:
what does that mean concretely?

Reply:
do we have any examples PR from previous releases?

Reply:
Yup documented in the corresponding section in release-tasks.md

Reply:
(will add links from issue tempalte to doc in a follow-up PR to avoid more broken links that are hard to verify in this PR)